automation/jleechanorg_pr_automation/tests/test_workflow_specific_limits.py

```
@@ -220,6 +220,282 @@ def test_count_fixpr_ignores_impostors(self):
+        now = datetime.now(timezone.utc)
+        comment_time = now - timedelta(hours=23)
+
+        comments = [
+            {
+                "body": "<!-- fixpr-run-automation-commit:codex:abc123 -->",
+                "author": {"login": "test-automation-user"},
+                "createdAt": comment_time.isoformat(),
+            },
+        ]
+
+        count = self.monitor._count_workflow_comments(comments, "fixpr")
```

Ruff (0.14.14)

[error] Private member accessed: `_count_workflow_comments`

(SLF001)